tests/test.bal

```diff
@@ -481,6 +482,7 @@ function testDescribeLimits() returns error? {
 
 @test:AfterSuite
 function testDeleteTable() returns error? {
+    runtime:sleep(20);
```

Comment:
Why do we need to wait for 20 ms before starting delete? Seems like `20ms` is subjective.

Reply:
There is a delay after creating and updating a table from DynamoDB side. When we try to delete table in the after suite test function, it fails because of the delay of updating table. To avoid that we need to put a sleep before the delete operation. Anyway the `20 second` delay is subjective.

Reply:
Shall we apply https://gist.github.com/abeykoon/09d3584039d285e29b0add84e13f5f40

Reply:
+1
The changes have been added.